app/controllers/welcome_controller.rb

```diff
@@ -1,4 +1,5 @@
 class WelcomeController < ApplicationController
   def index
+     @students = Student.sort_alphabetically_by_last_name
```

Comment:
This should be on the student controller. The welcome page is public and shouldn't show student data.